pkg/cmd/run/run.go

```diff
@@ -75,6 +78,7 @@ func NewCmdRun(o *Options) *cobra.Command {
 	bootstrapFlags := namedFlagSets.FlagSet("bootstrap")
 	bootstrapFlags.BoolVar(&o.BootstrapCRDs, "crd", true, "if set, the operator will attempt to install/update the CRDs before starting up.")
 	bootstrapFlags.StringVar(&o.BootstrapSpicedbsPath, "bootstrap-spicedbs", "", "set a path to a config file for spicedbs to load on start up.")
+	bootstrapFlags.StringVar(&o.WatchNamespaces, "watch-namespaces", "", "set a comma-separated list of namespaces to watch for CRDs.")
 	debugFlags := namedFlagSets.FlagSet("debug")
 	debugFlags.StringVar(&o.DebugAddress, "debug-address", o.DebugAddress, "address where debug information is served (/healthz, /metrics/, /debug/pprof, etc)")
 	o.ConfigFlags.AddFlags(namedFlagSets.FlagSet("kubernetes"))
```

Comment:
Could you pull the kubernetes flagset out into a named variable and add the namespace flag to it? see: `namedFlagSets.FlagSet("kubernetes")`

The `boostrap` flag set is a group of flags that control bootstrapping of the operator / spicedb itself by controlling whether it creates CRDs or pre-defined SpiceDBClusters on startup. I think namespaces fit better in the kubernetes group.